compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -528,18 +543,19 @@ trait ParallelTesting extends RunnerOrchestration { self =>
           else o
         }.mkString(JFile.pathSeparator)
 
+      val pageWidth = 80
       val flags1 = flags.copy(defaultClassPath = substituteClasspath(flags.defaultClassPath))
         .withClasspath(targetDir.getPath)
         .and("-d", targetDir.getPath)
+        .and("-pagewidth", pageWidth.toString)
 
-      val dummyStream = new PrintStream(new ByteArrayOutputStream())
-      val reporter = TestReporter.reporter(dummyStream, ERROR)
+      val reporter = TestReporter.reporter(System.out, ERROR)
```

Comment:
I would say we don't want to show all the logs in all situations as we don't show them by default when compiling with the latest compiler inside the same process. I think it would make sense to show the logs if the external process failed but no errors were parsed or if there's a mismatch between the expected number and positions of errors and the errors that we extracted from the logs

Reply:
Now errors are not printed in neg tests unless the test fails.